Packs/Gatewatcher-AionIQ/Integrations/GCenter/GCenter.py

```diff
@@ -0,0 +1,1353 @@
+            )
+            return response.json()
+        else:
+            raise GwAPIException(
+                f"Send powershell file {filename} on GCenter {self.ip}: [FAILED]",
+                response.text, response.status_code, response.reason
+            )
+
+
+def test_module(client: GwClient) -> str:  # noqa: E501
+    """Tests API connectivity and authentication command.
+
+    Args:
+        client: Client to interact with the GCenter.
+
+    Returns:
+        'Authentication successful' when the GCenter connection works.
+        'Authentication error' when the GCenter connection doesn't works.
+    """
+    if client.is_authenticated():
+        return "ok"
+    else:
+        return "Authentication error, please check ip/user/password/token: [ERROR]"
+
+
+def gw_list_alerts(client: GwClient, args: Optional[Dict[Any, Any]]) -> CommandResults:  # noqa: E501
+    """Get the latest elasticsearch alerts sorted by date in
+    descending order (most recent first in the list) command.
+
+    Args:
+        client: Client to interact with the GCenter.
+        args: Command arguments.
+
+    Returns:
+        CommandResults object with the "GCenter.Alert.List" prefix.
+    """
+    result = client.list_alerts()
+    readable_result = tableToMarkdown("Elasticsearch alerts list", result)
+    return CommandResults(
+        readable_output=readable_result,
+        outputs_prefix="GCenter.Alert.List",
+        outputs_key_field="id",
+        outputs=result,
+        raw_response=result
+    )
+
+
+def gw_get_alert(client: GwClient, args: Optional[Dict[Any, Any]]) -> CommandResults:  # noqa: E501
+    """Get an elasticsearch alert by uid command.
+
+    Args:
+        client: Client to interact with the GCenter.
+        args: Command arguments.
+
+    Returns:
+        CommandResults object with the "GCenter.Alert.Single" prefix.
+    """
+    result = client.get_alert(
+        uid=args.get("uid")  # type: ignore
+    )
+    readable_result = tableToMarkdown("Elasticsearch alert entry", result)
+    return CommandResults(
+        readable_output=readable_result,
+        outputs_prefix="GCenter.Alert.Single",
+        outputs_key_field="id",
+        outputs=result,
+        raw_response=result
+    )
+
+
+def gw_add_malcore_list_entry(client: GwClient, args: Optional[Dict[Any, Any]]) -> CommandResults:  # noqa: E501
+    """Add malcore whitelist/blacklist entry command.
+
+    Args:
+        client: Client to interact with the GCenter.
+        args: Command arguments.
+
+    Returns:
+        CommandResults object with the "GCenter.Malcore" prefix.
+    """
+    result = client.add_malcore_list_entry(
+        ltype=args.get("type"),  # type: ignore
+        sha256=args.get("sha256"),  # type: ignore
+        comment=args.get("comment", "added by cortex"),  # type: ignore
+        threat=args.get("threat", "unknown")  # type: ignore
+    )
+    readable_result = tableToMarkdown("Malcore whitelist/blacklist entry", result)
```

Comment:
```suggestion
    """
    list_type = args.get("type")
    result = client.add_malcore_list_entry(
        ltype=list_type,  # type: ignore
        sha256=args.get("sha256"),  # type: ignore
        comment=args.get("comment", "added by cortex"),  # type: ignore
        threat=args.get("threat", "unknown")  # type: ignore
    )
    readable_result = tableToMarkdown(f"Malcore {list_type}list entry", result)
```
I think it would be better to mention in the readable output if the entry eas added to a black list or to a white list. 
(You can change it in all the black\white list commands).

Reply:
I made the changes for the `gw_add_malcore_list_entry` and the `gw_add_dga_list_entry` command.